src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+
+      <ul class="stars stars--1">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+
+      <ul class="stars stars--2">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+
+      <ul class="stars stars--3">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+
+      <ul class="stars stars--4">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+
+      <ul class="stars stars--5">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
```

Comment:
Consider using more semantic HTML tags instead of `<ul>` and `<li>` for the star ratings. Semantic tags help improve accessibility and SEO. For example, you could use `<section>` or `<article>` for grouping content and `<span>` or `<div>` with appropriate ARIA roles for individual stars.